lte/gateway/c/core/oai/tasks/ngap/ngap_amf_nas_procedures.c

```diff
@@ -622,6 +608,27 @@ void ngap_handle_conn_est_cnf(
   ie->value.choice.RAN_UE_NGAP_ID = ue_ref->gnb_ue_ngap_id;
   ASN_SEQUENCE_ADD(&out->protocolIEs.list, ie);
 
+  // Ue aggregate maximum bit rate
```

Comment:
Can you put a comment in the summary reason for the change. Seems like its aligned as per the recommendation in specs.

Reply:
Sure, I'll update the summary. These changes are done to align the sequence according to 3gpp specs.

Reply:
Sure, I'll add in the summary, these changes are part of refactoring to align the sequence according to 3gpp spec.